dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/model/email/Email.kt

```diff
@@ -0,0 +1,29 @@
+package org.dataland.datalandcommunitymanager.model.email
+
+import com.mailjet.client.transactional.TransactionalEmail
```

Comment:
Another question. When are emails sent? Only when using the frontend or also when using the swaggerUi?

Reply:
Emails are always sent when a user posts a bulk data request.
It doesn't matter to the community manager where the request comes from.

Reply:
Ok then i have to check why it does not work in my local setup

Reply:
PRobably because you haven't set the env NOTIFICATION_RECEIVERS_BULK_DATA_REQUEST to your email address.

If you set your email address in that env, you will receive the mail when posting a bulk data request.

Reply:
I've set this and also the CC one. But got no email.

Reply:
I just tried again. FOr me it works.  And it also works on dev.   So something must be wrong about your environment.
Some checks you could do:
- Have you restarted your stack in a shell that was opened AFTER you have set the new envs?
- Can you see your mail if you "echo $NOTIFICATION_RECEIVERS_BULK_DATA_REQUEST" in a bash?
- What do the logs of the community manager say if you post a bulk data request? If the mail could not be sent, there will be an error there.    
- Are the mailjet credentials set in your envs?

Reply:
Thanks for the good tipps. My Mailjet secret wasn't up to date. Now it works